defactor-com/ui-kit
# feat(table): add filters

## Add filters

### What does this PR do?

- Resolve #55 
- Resolve #128

### Steps to test

1. Run project
2. Check changes
 

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/a45db8ae-3a25-447b-a536-d633091b4331)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/97f91472-8425-46c6-aeb8-b99d9115f744)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/b9052431-5ccd-4967-b3c7-274216e44c01)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/45c23265-924e-4a10-abea-06adf2a3dcb9)


#### CheckList
